mako/__init__.py

```diff
@@ -4,5 +4,5 @@
 # This module is part of Mako and is released under
 # the MIT License: http://www.opensource.org/licenses/mit-license.php
 
+from ._version import version as __version__
 
-__version__ = '1.0.2'
```

Comment:
sure you can - setup.py egg_info.
AFAIK it is better to use virtual env instead of manipulating PYTHONPATH (like tox).